lib/status/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+
+var status = angular.module('guStatus', []);
+
+status.directive('guStatus', function statusDirective(){
+  return {
+    restrict: 'E',
+    replace: true,
```

Comment:
Actually I read up on it.  It's not depreciated its just discouraged.  This is why I'm using custom attributes like `variant`.  Take for example the grid; we are just adding classes to custom elements in the link function. This feels wrong. If you don't add the class to the container element then the grid will not work so you need to replace and use a template which feels a lot cleaner.